apps/vanilla/src/pages/blackJack/models/player.ts

```diff
@@ -0,0 +1,51 @@
+import { Subscriber } from '../type/subscriber';
+
+import { Publisher } from './publisher';
+const WIN_SCORE = 21;
+
+/** Player with publisher and subscriber func. */
+export class Player extends Publisher<number> implements Subscriber<number> {
+	/** All dice results for one player. */
+	public readonly diceResults: number[] = [];
```

Comment:
Class should not expose mutable entities. Try to make this property `private`
 ```suggestion
	/** All dice results for one player. */
	private readonly diceResults: number[] = [];
```
If this property is used outside we should adjust the code. For instance:
```diff
- containerPlayer.innerHTML = player.diceResults.join(' ');
+ containerPlayer.innerHTML = player.printResult();
```